### PR TITLE
Update GraphQL API getting started to gold standard

### PR DESCRIPTION
**Pull request by richardmcclellan**
https://app.asana.com/0/1158799035447930/1169671161142084

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/graphqlapi/fragments/android/getting-started.md [hunk 1]
```diff
@@ -1,41 +1,17 @@
-You can integrate with Amplify framework using the following steps:
+> ### Prerequisites
+> * An Android project targeting Android API level 16 (Android 4.1) or above.
```
Comment (jpignata) [inline docs/lib/graphqlapi/fragments/android/getting-started.md:2]: Can we be consistent with punctuation on bullets? I'd prefer without, but "both" is the wrong answer. Also! I wonder if we can link to Google's Android documentation here directly to their getting started steps.

File: docs/lib/graphqlapi/fragments/android/getting-started.md [hunk 2]
```diff
@@ -1,41 +1,17 @@
...
+> * [Install and configure](~/cli/start/install.md) the Amplify CLI
 
-1. Setup the API endpoint and authentication information in the client side configuration.
-2. Generate Java Model classes from the API schema.
-3. Write app code to run queries, mutations and subscriptions.
+## GraphQL API with Amplify
 
-## Set up your backend
+The Amplify GraphQL API category provides a solution for making HTTP requests to GraphQL endpoints.  It supports [AWS AppSync](https://aws.amazon.com/appsync/) or any other GraphQL backend.
```
Comment (jpignata) [inline docs/lib/graphqlapi/fragments/android/getting-started.md:7]: This actually isn't true at the moment. We only support AWS AppSync, I believe.

Comment (jpignata) [inline docs/lib/graphqlapi/fragments/android/getting-started.md:7]: "Amplify GraphQL API category" isn't the right terminology here probably. We should say "The Amplify API category provides a solution for making HTTP requests to REST and GraphQL endpoints. For GraphQL, it supports ..." something like that.

File: docs/lib/graphqlapi/fragments/android/getting-started.md (hunk 1 above)
Comment (brene) [reply docs/lib/graphqlapi/fragments/android/getting-started.md:2]: Same. prefer without punctuation on bullets

File: docs/lib/graphqlapi/fragments/android/getting-started.md (hunk 2 above)
Comment (brene) [reply docs/lib/graphqlapi/fragments/android/getting-started.md:7]: @jpignata - I was just chatting with @nikhil-dabhade about this and he pointed that the JS libraries do support other GQL backends as well. Is this Android-specific?

Comment (jpignata) [reply docs/lib/graphqlapi/fragments/android/getting-started.md:7]: Yes, JavaScript does, but neither iOS or Android do at the moment, I believe.

File: docs/lib/graphqlapi/fragments/android/getting-started.md (hunk 1 above)
Comment (richardmcclellan) [reply docs/lib/graphqlapi/fragments/android/getting-started.md:2]: Good suggestions!  I will:
 - remove punctuation
 - add link to Google Android docs

File: docs/lib/graphqlapi/fragments/android/getting-started.md (hunk 2 above)
Comment (richardmcclellan) [reply docs/lib/graphqlapi/fragments/android/getting-started.md:7]: I actually copied this from the JS page, so good to know that iOS and Android don't support it yet :) Will remove.

Comment (richardmcclellan) [reply docs/lib/graphqlapi/fragments/android/getting-started.md:7]: Makes sense!

**Pull request merged**